synapse/lib/httpapi.py

```diff
@@ -496,7 +496,9 @@ async def get(self):
 
 class WatchSockV1(WebSocket):
     '''
-    A web-socket based API endpoint for distributing cortex events.
+    A web-socket based API endpoint for distributing cortex tag events.
+
+    Deprecated.
```

Comment:
can we add a `s_common.deprecated` marker as well?

Reply:
Yes we can :+1: